sagemaker-fundamentals/create-training-job/create_training_job_encrypt_static_assets.ipynb

```diff
@@ -0,0 +1,1227 @@
+{
```

Comment:
<p>Can you edit this image to remove "master"? You can use "KMS key" for specific KMS reference, or "root key" for more generic encryption.</p><p> </p><p>Nit: need a period after "envelope encryption".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I replaced the "Master key" by "KMS key", but I don't know the font style and size of "Data key" and "Data" on the image and I only have an elementary image editor on my computer, so the edited image looks a bit funny.</p>